### DISCUSSION TITLE
No SEARCH FIELD in MOBILE VIEW

### DISCUSSION THREAD

**Discussion by MichaelFranziskus** [mastodon/mastodon#22649] [Q&A] [answered]
Hi folks,
I've started to use mastodon.social in the sidebar of my vivaldi browser.
It seems like it is the **responsive mobile view** of the page.

Unfortunatly I **can not finde the search field**:
![image](https://user-images.githubusercontent.com/36480894/209077791-7e35e51c-27bf-4bfa-ba40-591eef8af742.png)

Maybe someone can say me where or can create a bug-ticket.

KR MichaelFranziskus

**Comment by rbairwell** [comment, marked as answer]
On mobile view, the search box appears once the hashtag "#" (for "Explore") on the right has been clicked.

**Reply by odoketa** [reply]
Definitely think this is a UX issue.

**Reply by rbairwell** [reply]
It's already been requested as an improvement in https://github.com/mastodon/mastodon/issues/19768#issuecomment-1320890365.

**Comment by olderwoman** [comment]
FWIW the same problem (and behavior when # is chosen) appears on the desktop if the monitor has a narrow aspect ratio.
